float32[] cmd1
float32[] cmd2
float32[] cmd3
float32[] cmd4
float32[] cmd5
float32[] cmd6
float32 velocity
float32 acceleration
---
bool success